### PR TITLE
Adding a backend. What is commented out is not ready yet.

### PR DESCRIPTION
**Pull request by baklanovairina**

### CODE REVIEW

File: src/models/cost.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+
+const {Schema} = mongoose;
```
Comment (usatovaDasha) [inline src/models/cost.js:3]: spaces

File: src/models/cost.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+const costScheme = new Schema({
+    id: String,
```
Comment (usatovaDasha) [inline src/models/cost.js:6]: what is it id?

File: src/modules/controllers/cost.controller.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+const Cost = require("../../models/cost");
+
+// let costs = [];
```
Comment (usatovaDasha) [inline src/modules/controllers/cost.controller.js:3]: remove all comments

File: src/modules/controllers/cost.controller.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+module.exports.getAllCosts = (req, res, next) => {
+  Cost.find().then(result => {
+    res.status(200).send({data: result});
+  });
+};
+
+module.exports.createNewCost = (req, res, next) => {
+  const body = req.body; 
+  if (body.hasOwnProperty('place') && body.hasOwnProperty('money')) {
+    body.id = Math.random().toString(36).substring(2) + (new Date()).getTime().toString(36);
```
Comment (usatovaDasha) [inline src/modules/controllers/cost.controller.js:14]: id create DB, why you need this line?

File: src/modules/controllers/cost.controller.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+module.exports.getAllCosts = (req, res, next) => {
+  Cost.find().then(result => {
+    res.status(200).send({data: result});
+  });
+};
+
+module.exports.createNewCost = (req, res, next) => {
+  const body = req.body; 
+  if (body.hasOwnProperty('place') && body.hasOwnProperty('money')) {
```
Comment (usatovaDasha) [inline src/modules/controllers/cost.controller.js:13]: date no need?

File: src/modules/controllers/cost.controller.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+//   } else {
+//     res.status(422).send('Error! Params not correct');
+//   }
+// };
+
+module.exports.deleteCost = (req, res, next) => {
+    const body = req.body;
+    if (body.hasOwnProperty('id')) {
+      Cost.deleteOne(
+      { id: body.id },
```
Comment (usatovaDasha) [inline src/modules/controllers/cost.controller.js:42]: use uniq id in collection

File: src/modules/controllers/cost.controller.js (hunk 5 above)
Comment (baklanovairina) [reply src/modules/controllers/cost.controller.js:13]: No, because there is no field for it in the task. It generated itself, but it edit.

Comment (usatovaDasha) [reply src/modules/controllers/cost.controller.js:11]: and on which side is it generated? it does not need to be saved in the database?

Comment (baklanovairina) [reply src/modules/controllers/cost.controller.js:11]: it is saved and even overwritten in the database

**Pull request merged**